commands.txt

```diff
@@ -0,0 +1,8 @@
+if [ `uname` = 'Darwin' ]; then echo ERROR: coverage feature not implemented && exit 0; fi
+if ![ -f sonar-project.properties ]; then echo "ERROR: No sonar-project.properties file detected." && exit 0; fi
+export SD_CURL_CMD_WRAPPER="$(echo 'sd-step exec --pkg-version 7.54.1 core/curl')"
+export SD_UNZIP_CMD_WRAPPER="$(echo 'sd-step exec --pkg-version 6.0 core/unzip')"
```

Comment:
Instead of `export NAME="$(echo VALUE)"`, can these 2 exports just be `export NAME="VALUE"`?

Reply:
yes, good point